www/widgets/rt.js

```diff
@@ -31,6 +31,46 @@ rt_constants.ZOOM = 1.0;
 rt_constants.TKELEMENT = null;
 rt_constants.URLS = {};
 
+// This function exists because different browsers draw text in
+// different positions even given the same textBaseline. The
+// implementation borrows a technique from [1], which draws a 30px "F"
+// on a canvas and measures its position to compute an offset that is
+// then scaled by font size and applied to any create_text call. A
+// 30px serif font is used because that is used for the new tab "+"
+// button, whose alignment is most visible.
+//
+// [1]: https://pqina.nl/blog/cross-browser-alignment-of-the-canvas-filltext-draw-call/
```

Comment:
This is a pretty extreme hack to avoid this problem.

Let me go talk to the canvas team next week and see what's up.